README.md

```diff
@@ -1,12 +1,12 @@
 # Arti-rest
 
-If you ever wanted to use tor in a mobile app, up to now you always had to
-install a tor-proxy alongside your mobile app. 
-Then you could use this tor-proxy to pass all your requests through.
-`Arti-rest` is a library that can be directly integrated into an Android
-or iOS app if all you want to do is a simple REST request over the Tor network.
+> :warning: **Warning: arti-rest is not secure in all situations** Arti-rest modifies several core parts of `arti` and therefore does not provide the same guarantees that arti and the stock Tor client would. Security depends on whether these weakened guarantees are acceptable in your situation.
 
-Arti-wrapper to use REST over Tor:
+Arti-rest is a simple wrapper around [arti](https://gitlab.torproject.org/tpo/core/arti) that leverages arti to enable mobile apps to easily make anonymous REST requests via the Tor network. Arti-rest makes two changes on top of arti. First, arti-rest provides a simple wrapper for REST requests, to make calling HTTP(S) API endpoints easier. Second, arti-rest provides the option to use a customized, and therefore potentially much smaller, consensus. Using a customized consensus is less secure than using the original Tor consensus. It therefore depends on your application whether this reduction is worth while.
+
+Arti-rest is written in Rust and can therefore be compile into native libraries that is easy to integrate into mobile applications. All credits for enabling this approach go to the authors of [arti](https://gitlab.torproject.org/tpo/core/arti). Using a native library ensures that arti-rest can be bundled with Android and iOS applications, rather than relying on external Tor proxies that must be installed separately.
```

Comment:
```suggestion
Arti-rest is written in Rust and can therefore be compiled into native libraries that are easy to integrate into mobile applications. All credits for enabling this approach go to the authors of [arti](https://gitlab.torproject.org/tpo/core/arti). Using a native library ensures that arti-rest can be bundled with Android and iOS applications, rather than relying on external Tor proxies that must be installed separately.
```